EIPS/eip-7702.md

```diff
@@ -8,7 +8,7 @@ status: Draft
 type: Standards Track
 category: Core
 created: 2024-05-07
-requires: 2718, 2930
+requires: 2718, 2930, 2929
```

Comment:
@SamWilsn 2930 requires 2929 so this is redundant I think

Reply:
hmm 2930 also requires 2718

Reply:
Yeah, that's fair. I mentioned it explicitly because I linked to it. There aren't any hard rules on transitive `requires` :rofl: